### ISSUE TITLE
Timeline redactor stop working

### ISSUE THREAD

**Issue by AMSCRIPTOR** [dialogic-godot/dialogic#2150]
## The problem

**Describe the bug**
Dont know what Im doing wrong, I change nothing in subsystem files, but text redactor (and visual redactor too) suddenly stop working: I cant write new text, cant change redactor type, it just dont click or scroll. I still can create new timelines and characters, but I cant write new text or correct oldest.1
Restarting the engine or computer did not solve the problem.
![image](https://github.com/dialogic-godot/dialogic/assets/117640126/f90d6244-c1e9-4ffe-91dd-ee901b4b36ab)
**System (please complete the following information):**
 - OS: [e.g. Windows]
 - Godot Version: [e.g. 4.2 stable]
 - Dialogic Version: [e.g. 2]

**Comment by CakeVR** [comment]
You can open the older timelines in your own editor to check.
Odds are, they are really empty.

**Comment by AMSCRIPTOR** [comment]
No, timelines are not empty, they all have text, but I can't edit it and don't understand why
literally two days ago everything was fine and I didn’t change anything :(
![image](https://github.com/dialogic-godot/dialogic/assets/117640126/b69fdd1c-ded2-447b-8bbf-0590ad2e5ccd)

**Comment by CakeVR** [comment]
Try the latest commit on the `main` branch.
Do you have any errors?

**Comment by Jowan-Spooner** [comment]
Will close this as we cannot recreate this.
